app-backend/app/src/main/resources/application.conf

```diff
@@ -41,6 +41,12 @@ auth0 {
   # Auth0 Bearer with proper permissions for Management API
   bearer = ""
   bearer = ${?AUTH0_MANAGEMENT_BEARER}
+
+  managementClientId = ""
+  managementClientId = ${?AUTH0_MANAGEMENT_CLIENT_ID}
```

Comment:
That's correct. I had to enable the APIs section (under advanced account settings). I believe that the management API authorization is done through there. I set credential scopes to only allow CRUD of device_tokens. I do think we'll have to do that for each account and then thread those credentials through Terraform.